src/fix.rs

```diff
@@ -0,0 +1,77 @@
+use diesel;
+use diesel::prelude::*;
+use diesel::types::{Integer, VarChar};
+
+use github::{update_issue, update_pr};
+
+use DB_POOL;
+
+pub fn fix() {
+    for (repo, number) in get_duplicate_issues() {
+        delete_duplicate_issues(&repo, number);
+        update_issue(&repo, number).expect("Failed to update issue");
```

Comment:
Lol yeah, it might be better to just merge it now (and it's not like this script will be used repeatedly 🤞). I'll probably have time to refactor things later, but no guarantee on when that will actually be.